frontend/src/components/ui/chat/ChatTextBox.module.css

```diff
@@ -0,0 +1,20 @@
+.chattextbox {
```

Comment:
For the naming can help split the word so more readable? like `chat_textbox`. Thanks, same applies to the below too 😄

Reply:
Thanks for this, did not notice this.